andromeda/dataset.py

```diff
@@ -50,7 +50,7 @@ def exists(self):
         return os.path.exists(self.get_path())
 
     def read_dataframe(self):
-        return pd.read_csv(self.get_path())
+        return pd.read_csv(self.get_path(), keep_default_na=False)
```

Comment:
Maybe comment saying  `# Produces invalid JSON if not set to False` .

Reply:
I added a comment explaining the reasoning for the`keep_default_na` flag.